packages/api-explorer/src/components/DocMethodSummary/DocMethodSummary.tsx

```diff
@@ -37,31 +35,29 @@ interface DocMethodSummaryProps {
   method: IMethod
 }
 
-export const DocMethodSummary: FC<DocMethodSummaryProps> = ({ method }) => {
-  return (
-    <SummaryCard>
-      <CardContent>
-        <Space align="start" between>
-          <MethodBadge
-            type={method.httpMethod}
-            textAlign="center"
-            minWidth="3.7625rem"
-          >
-            {method.httpMethod.toUpperCase()}
-          </MethodBadge>
-          <Flex alignItems="start" flexDirection="column" flex="1" mx="small">
-            <ApixHeading as="h3" mb="0" pt="0">
-              {method.summary}
-            </ApixHeading>
-            <Space>
-              <DocRateLimited method={method} />
-              <DocActivityType method={method} />
-            </Space>
-            <Text variant="subdued">{method.endpoint}</Text>
-          </Flex>
-          <DocSummaryStatus method={method} />
-        </Space>
-      </CardContent>
-    </SummaryCard>
-  )
-}
+export const DocMethodSummary: FC<DocMethodSummaryProps> = ({ method }) => (
+  <Card maxWidth="50rem" borderColor="ui2">
+    <CardContent>
```

Comment:
SummaryCard was just setting `maxWidth` and customized `borderColor` both of which are support by simple property assignment.